doc/intro.xml

```diff
@@ -54,11 +54,17 @@ usage of this package.<P/>
 </Section>
 
 <Section Label="feedback">
-<Heading>Feedback</Heading>
+<Heading>Feedback and support</Heading>
 
-If you have any bug reports, feature requests, or suggestions about the
-package, please use our
-<URL Text="issue tracker">https://github.com/gap-packages/recog/issues</URL>.
+If you have any bug reports, feature requests, or suggestions, then please
+tell us via the <URL Text="issue tracker on
+GitHub">https://github.com/gap-packages/recog/issues</URL>. <P/>
+
+In addition, the <Package>recog</Package> package has a mailing list, at
+<URL>recog@gap-system.org</URL>, which can be used for holding discussions,
```

Comment:
Does GAPDoc handle emails like that? Think is, `recog@gap-system.org` is not a valid URL, but `mailto: recog@gap-system.org` is. So perhaps this needs to be `<URL Text="recog@gap-system.org"> recog@gap-system.org</URL>`?

Reply:
Yes it wasn't working.